### PR TITLE
Updated helm chart to support PowerStore metrics

### PR DESCRIPTION
**Pull request by tdawe**
#### Is this a new chart?

No

#### What this PR does / why we need it:

Updates the helm chart for Observability to support PowerStore metrics.

#### Which issue(s) is this PR associated with:

- https://github.com/dell/karavi-observability/issues/48

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Chart Version bumped
- [x] Variables are documented in the chart README.md
- [ ] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. Looks good. There are no editorial issues.

File: charts/karavi-observability/templates/karavi-metrics-powerflex-service-account.yaml [hunk 1]
```diff
@@ -35,3 +37,5 @@ roleRef:
   kind: ClusterRole
   name: {{ .Release.Name }}-metrics-powerflex-controller
   apiGroup: rbac.authorization.k8s.io
+
+{{ end }}
```
Comment (ianbibby) [inline charts/karavi-observability/templates/karavi-metrics-powerflex-service-account.yaml:41]: Add new line.

File: charts/karavi-observability/templates/karavi-metrics-powerflex.yaml [hunk 2]
```diff
@@ -74,3 +76,5 @@ spec:
           name: karavi-metrics-powerflex-configmap
       restartPolicy: Always
 status: {}
+
+{{ end }}
```
Comment (ianbibby) [inline charts/karavi-observability/templates/karavi-metrics-powerflex.yaml:80]: Add new line.

File: charts/karavi-observability/templates/karavi-metrics-powerstore-service-account.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Release.Name }}-metrics-powerstore-controller
+    namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: {{ .Release.Name }}-metrics-powerstore-controller
+  apiGroup: rbac.authorization.k8s.io
+
+{{ end }}
```
Comment (ianbibby) [inline charts/karavi-observability/templates/karavi-metrics-powerstore-service-account.yaml:38]: Add new line.

File: charts/karavi-observability/templates/karavi-metrics-powerstore.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          items:
+            - key: tls.crt
+              path: cert.crt
+      - name: karavi-metrics-powerstore-configmap
+        configMap:
+          name: karavi-metrics-powerstore-configmap
+      restartPolicy: Always
+status: {}
+
+{{ end }}
```
Comment (ianbibby) [inline charts/karavi-observability/templates/karavi-metrics-powerstore.yaml:80]: Add new line.

File: charts/karavi-observability/templates/karavi-topology-service-account.yaml [hunk 5]
```diff
@@ -29,3 +31,5 @@ roleRef:
   kind: ClusterRole
   name: {{ .Release.Name }}-topology-controller
   apiGroup: rbac.authorization.k8s.io
+
+{{ end }}
```
Comment (ianbibby) [inline charts/karavi-observability/templates/karavi-topology-service-account.yaml:35]: Add new line.

File: charts/karavi-observability/templates/karavi-topology.yaml [hunk 6]
```diff
@@ -64,3 +66,5 @@ spec:
           mountPath: "/etc/config"
       restartPolicy: Always
 status: {}
+
+{{ end }}
```
Comment (ianbibby) [inline charts/karavi-observability/templates/karavi-topology.yaml:70]: Add new line.

**Review by ianbibby — APPROVED** [review]
LGTM.  Just comments for the missing new lines which or may not be an issue.

File: charts/karavi-observability/templates/karavi-metrics-powerflex-service-account.yaml (hunk 1 above)
Comment (tdawe) [reply charts/karavi-observability/templates/karavi-metrics-powerflex-service-account.yaml:41]: Fixed

File: charts/karavi-observability/templates/karavi-metrics-powerflex.yaml (hunk 2 above)
Comment (tdawe) [reply charts/karavi-observability/templates/karavi-metrics-powerflex.yaml:80]: Fixed

File: charts/karavi-observability/templates/karavi-metrics-powerstore-service-account.yaml (hunk 3 above)
Comment (tdawe) [reply charts/karavi-observability/templates/karavi-metrics-powerstore-service-account.yaml:38]: Fixed

File: charts/karavi-observability/templates/karavi-metrics-powerstore.yaml (hunk 4 above)
Comment (tdawe) [reply charts/karavi-observability/templates/karavi-metrics-powerstore.yaml:80]: Fixed

File: charts/karavi-observability/templates/karavi-topology-service-account.yaml (hunk 5 above)
Comment (tdawe) [reply charts/karavi-observability/templates/karavi-topology-service-account.yaml:35]: Fixed

File: charts/karavi-observability/templates/karavi-topology.yaml (hunk 6 above)
Comment (tdawe) [reply charts/karavi-observability/templates/karavi-topology.yaml:70]: Fixed

**Review by alikdell — APPROVED** [review]

**Pull request merged**